kuksa-sdk/build.gradle.kts

```diff
@@ -74,3 +92,17 @@ protobuf {
         }
     }
 }
+
+publishing {
```

Comment:
clashes with #6 
depending on who get's merged first, the other one has to do some adaptions

Reply:
I will probably fix these :)